livebooks/exercises/A_intro.livemd

```diff
@@ -0,0 +1,195 @@
+# A: intro
+
+```elixir
+Logger.configure(level: :info)
+
+Mix.install([
+  :membrane_core,
+  {:workshop_elixir_conf_us_2024, path: Path.join(__DIR__, "../..")},
+  {:membrane_vpx_plugin, "~> 0.1.1"},
+  {:membrane_webrtc_plugin, "~> 0.19.0"},
+  {:membrane_camera_capture_plugin, "~> 0.7.2"},
+  {:membrane_h264_ffmpeg_plugin, "~> 0.31.6"},
+  {:membrane_h26x_plugin, "~> 0.10.2"},
+  {:membrane_sdl_plugin, "~> 0.18.2"},
+  {:membrane_mp4_plugin, "~> 0.35.1"},
+  {:membrane_file_plugin, "~> 0.17.0"},
+  {:membrane_matroska_plugin, "~> 0.6.0"},
+  {:membrane_ffmpeg_swscale_plugin, "~> 0.16.1"},
+  :membrane_realtimer_plugin,
+  {:kino, "~> 0.13.1"},
+  :membrane_opus_plugin,
+  {:membrane_raw_video_format, "~> 0.4.0", override: true},
+  {:membrane_vp8_format, "~> 0.5.0", override: true},
+  {:membrane_vp9_format, "~> 0.5.0", override: true},
+  {:unifex, "~> 1.2", override: true},
+  {:nx, "~> 0.7.0"},
+  {:exla, "~> 0.7.0"},
+  {:ortex, "~> 0.1.9"}
+])
```

Comment:
- for consistency, let's provide the version requirements everywhere
- why these overrides?
- there are some deps that aren't needed in this particular livebook, for example membrane_webrtc_plugin
- maybe we could add these as dependencies to the mix project (`workshop_elixir_conf_us_2024`) to avoid repeating them in all livebooks?

Reply:
1. OK
2. Because otherwise there would be version conflicts related to changes in VPx plugin
3. Yes, but now `Mix.install` will run only once for all livebooks. If there is any change between livebooks deps, `Mix.install/1` will run from the begging in each livebook and it will take a lot of time, since it compiles `ortex` and `exla`
4. Do you think that having all libraries used in the exercises listed in `Mix.install/1` is somehow bad?

Reply:
2. Can we resolve that somehow? Is that in progress or is there any issue regarding that?
4. Now it's quite misleading as we have some unnecessary deps there. But worse thing is that whenever someone wants to change the deps or their versions, they have to do that in 3 places

Reply:
Ok, I will move it to the deps of `:workshop...` project